File: readability/readability.c
#include <cs50.h>
#include <stdio.h>
#include <string.h>

int count_letters(string text);
int count_words(string text);
int count_sentences(string text);

int main(void)
{
    string text;
    int letters, words, sentences;


    text = get_string("Text: ");


    letters = count_letters(text);
    printf("letters: %d\n",letters);
    words = count_words(text);
    printf("words: %d\n",words);
    sentences = count_sentences(text);
    printf("sentences: %d\n",sentences);




    printf("Text: %s\n",text);

}

int count_letters(string text)
{
    int counter = 0;
    for( int i = 0; i < strlen(text); i++ )
    {
        if( (text[i] >= 65 && text[i] <= 90) || (text[i] >= 97 && text[i] <= 122 )  )
        {
            counter++;
        }
    }
    return counter;
}

int count_words(string text)
{
    int counter = 1;
    for( int i = 0; i < strlen(text); i++ )
    {
        if( text[i] == 32 )
        counter++;
    }
    return counter;
}

int count_sentences(string text)
{
    int counter = 0;
    for( int i = 0; i < strlen(text); i++ )
    {
        if( text[i] == 33 || text[i] == 46 || text[i] == 63 )
        {
            counter++;
        }
    }
    return counter;
}

File: sentimental-readability/readability.py
# TODO
from cs50 import get_string

text = get_string("Text: ")

def count_letters(text):
    counter = 0
    for i in text:
        if text[i].islower() | text[i].isupper:
            counter += 1
    return counter

letters = count_letters(text)

def count_words(text):
    counter = 1
    for i in text:
        if text[i] == 32:
            counter += 1
    return counter
words = count_words(text)

def count_sentences(text):
    counter = 0
    for i in text:
        if text[i] in {33, 46,63}:
            counter += 1
    return counter
sentences = count_sentences(text)

L = (letters / words) * 100
S = (sentences / words) * 100

index = round((0.0588 * L) - (0.296 * S) - 15.8)

print(f"Text: {text}")

if (index < 1):
    print("Before Grade 1")
    elif (indes)

